### DISCUSSION TITLE
Drawing wires in non-90 degree angles

### DISCUSSION THREAD

**Discussion by mengstr** [hneemann/Digital#997] [Ideas]
Every now and then I draw wires that aren't straight horizontal or vertical. Currently I do that by just drawing a short stub and then dragging the end to its intended location since dragging can be done in arbitrary directions. 

But I haven't found any modifier key that allows new lines to be drawn at arbitrary angles directly. Is that something that might be implemented or does it already exists and I just haven't found it on my Mac keyboard?
<img width="443" alt="Screenshot 2022-07-26 at 11 09 41" src="https://user-images.githubusercontent.com/325326/180969566-61c85b43-6b91-4cb1-915e-845f71442183.png">

**Comment by hneemann** [comment]
When drawing a wire, the [D] key activates the diagonal mode and the [F] key flips the orientation of the rectangle.
At least on my computer this is the case :-)

**Reply by mengstr** [reply]
Nice!  Maybe I should have looked in the documentation instead of just trying modifier keys :)   The F key will also come in handy...
